### ISSUE TITLE
no case file download

### ISSUE THREAD

**Issue by digitaloutlets** [cytan299/etx125_electric_focuser#1]
The focuser_v4.stl download does not work

**Comment by cytan299** [comment]
What do you mean? More details please.

**Comment by digitaloutlets** [comment]
Hi cytan! When I press the download button for the stl file, it just opens up in the window but does not download the v4.stl file to my computer. I can't seem to download the focuser v4.stl file I need for Shapeways.
Thanks for your reply


________________________________

**Comment by cytan299** [comment]
One of the problems with github that it is not easy to download just one file. It's a lot easier to clone the entire project. So from the directory: 

https://github.com/cytan299/etx125_electric_focuser

Click on the green "Clone or download" button and you should get everything, including the stl file.

**Comment by digitaloutlets** [comment]
OMG! Thank you so much for your help. I did exactly how you said and cloned the whole file and it worked like a charm. I was also able to upload the individual stl file to Shapeways. I selected the 'professional plastic' option to be 3D printed and the cost is 70 dollars for the piece. Does that sound about right?
Again thank you for everything, this is a brilliant alternative for adapting the 1244 to work with the ETX-125 as a professionally made unit! Very pleased with the results so far!!

**Comment by cytan299** [comment]
Thanks! Hmmm, the cost seems to be *twice* what I had paid. See bom/bom.ods. Make sure you specified the conversion correctly (inches must be specified during upload)

<img width="475" alt="track" src="https://user-images.githubusercontent.com/13826223/42342614-39483a3c-805c-11e8-9d1f-a7f25350aca1.png">

BTW, this focuser design works great :) I've been using it to photograph Jupiter. See this that I took:

https://www.cloudynights.com/topic/398608-small-bore-challenge-jupiter-w-6-or-less/?p=8645735

with my ETX125.

**Comment by digitaloutlets** [comment]
The high cost was because I went with their 'Professional Plastic' option. 
The file was definitely uploaded properly with the conversion set correctly to inches, same specs in your pic (shown in cm).
I am really glad you shared this, and I saw this on cloudynights! This design was exactly what I needed!!
Bravo, that is a fantastic shot of Jupiter and the mini movie, nice job! BTW, I have a ETX-125PE
